nautobot/extras/apps.py

```diff
@@ -44,3 +44,10 @@ def convert_field_to_list_tags(field, registry=None):
         from nautobot.extras.health_checks import DatabaseBackend
 
         plugin_dir.register(DatabaseBackend)
+
+        # Register netutils jinja2 filters
+        from netutils.utils import jinja2_convenience_function
+        from django_jinja.library import filter, _register_function
+
+        for name, func in jinja2_convenience_function().items():
+            _register_function("filters", name=name, fn=func)
```

Comment:
Looks like this could be achieved without using a private function as follows:

```suggestion
        from django_jinja import library

        for name, func in jinja2_convenience_function().items():
            library.filter(name=name, fn=func)
```